### PR TITLE
chores(docs): Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

**Review by jeertmans — CHANGES_REQUESTED** [review]
Hi @kianmeng ! Many thanks for taking the time to review my code :)

There are two comments I have:
- Can you update the correction about « whether » ?
- Can you write a one line change that in the Unreleased/Chore section of the changelog ?

Feel free if you have any other question ;)

File: src/lib/check.rs [hunk 1]
```diff
@@ -416,7 +416,7 @@ pub struct Software {
     pub build_date: String,
     /// Name (should be `"LanguageTool"`)
     pub name: String,
-    /// Tell wether the server uses premium API or not
+    /// Tell weather the server uses premium API or not
```
[the remark this answers is not in the document]
Comment (kianmeng) [reply src/lib/check.rs]: Seemed you've added the change. :smile:

[the remark this answers is not in the document]
Comment (jeertmans) [reply src/lib/check.rs]: Yes, I wanted to "suggest changes" but I actually committed the fix ^^'

**Review by jeertmans — APPROVED** [review]

**Comment by jeertmans** [comment]
Merged! Many thanks @kianmeng ;-)

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**